cg/models/orders/samples.py

```diff
@@ -195,9 +195,16 @@ def str_to_int(cls, v: str) -> int | None:
 class PacBioSample(OrderInSample):
     _suitable_project = OrderType.PACBIO_LONG_READ
 
+    buffer: str
     capture_kit: str | None = None
+    container: ContainerEnum | None
```

Comment:
```suggestion
    container: ContainerEnum
```
I don't think it should be possible to send it as None. One of the values of the enum is "no container"